web/setups/serversyncscript.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright (c) 2017 University of Illinois, NCSA.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the
+ * University of Illinois/NCSA Open Source License
+ * which accompanies this distribution, and is available at
+ * http://opensource.ncsa.illinois.edu/license.html
+ */
+
+// Server side scripts to handle the client resquests
```

Comment:
How do we ensure that this script ONLY runs on one server within the PEcAn network?
How do we prevent someone who already has a host id from accidentally requesting one a second time?
How do we prevent a malicious 3rd party from setting up a loop that asks for billions of host ids?